src/sentry/runner/commands/devserver.py

```diff
@@ -227,16 +242,19 @@ def devserver(reload, watchers, workers, experimental_spa, styleguide, prefix, e
     # Make sure that the environment is prepared before honcho takes over
     # This sets all the appropriate uwsgi env vars, etc
     server.prepare_environment()
-    daemons += [("server", ["sentry", "run", "web"])]
+    daemons += [_get_daemon("server")]
 
     if styleguide:
-        daemons += [("storybook", ["./bin/yarn", "storybook"])]
+        daemons += [_get_daemon("storybook")]
 
+    skip_daemons = set(skip_daemons.split(",")) if skip_daemons else ()
     cwd = os.path.realpath(os.path.join(settings.PROJECT_ROOT, os.pardir, os.pardir))
 
     manager = Manager(Printer(prefix=prefix))
     for name, cmd in daemons:
-        manager.add_process(name, list2cmdline(cmd), quiet=False, cwd=cwd)
+        cmdline = list2cmdline(cmd)
+        if name not in skip_daemons:
```

Comment:
I think it would be a good idea to make this fatal. People might mess up the comma delimited part or make a typo.

Reply:
You mean if there is a name in `skip_daemons` that doesn't belong to an existing daemon? That's not what this check is doing, but it's a good idea -- I'll add one.